radix-engine-tests/tests/package.rs

```diff
@@ -425,36 +425,32 @@ fn name_validation_feature_set() {
         matches!(
             e,
             RuntimeError::ApplicationError(ApplicationError::PackageError(
-                PackageError::InvalidName(..)
+                PackageError::InvalidName { .. }
             ))
         )
     });
 }
 
 #[test]
-fn name_validation_function() {
+fn well_known_types_in_schema_are_validated() {
     // Arrange
     let mut test_runner = TestRunnerBuilder::new().build();
 
     let (code, mut definition) = PackageLoader::get("publish_package");
 
-    definition
+    let method_definition = definition
         .blueprints
         .values_mut()
         .next()
         .unwrap()
         .schema
         .functions
         .functions
-        .insert(
-            String::from("self"),
```

Comment:
`self` used to not be permitted but since we're no longer checking for `Ident` it's now permitted.